frontend/src/main/web/lib/components/RecentContributions.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ContributionChart from './ContributionChart';
+import DropDown from './DropDown';
+import FilterableMatrixTable from './FilterableMatrixTable';
+import {DateRanges} from '../constants/Options';
+import Actions from '../actions/UserMatrixActions';
+
+var RecentContributions = React.createClass(
+  {
+    render: function() {
+      var dateRange = this.props.dateRange, chart = null, matrix = null;
+
+      if(this.props.loading) {
+        chart = (
+          <a href="#" className="loader--large is-active">
+          <span className="loader__spinner">
+            <span></span>
+            <span></span>
+            <span></span>
+          </span>
+          </a>);
+      } else {
+        chart = (
+          <ContributionChart wordCountForEachDay={this.props.matrixForAllDays} dateRangeOption={this.props.dateRangeOption} />);
+
+        matrix =  (
+          <FilterableMatrixTable
+            wordCountForSelectedDay={this.props.wordCountsForSelectedDayFilteredByContentState}
+            wordCountForEachDay={this.props.wordCountsForEachDayFilteredByContentState}
+            fromDate={dateRange['fromDate']} toDate={dateRange['toDate']}
+            dateRangeOption={this.props.dateRangeOption}
+            selectedContentState={this.props.contentStateOption}
+            selectedDay={this.props.selectedDay}
```

Comment:
The last 3 could be done as `{..._.pick(this.props, ['dateRangeOption', 'selectedContentState', 'selectedDay'])}` - slightly shorter but not much of a difference with only 3 props.